eth2util/network.go

```diff
@@ -50,10 +50,16 @@ var (
 		GenesisForkVersionHex: "0x90000069",
 		GenesisTimestamp:      1655733600,
 	}
+	Holesky = Network{
+		ChainID:               17000,
+		Name:                  "holesky",
+		GenesisForkVersionHex: "0x00017000",
+		GenesisTimestamp:      1696000704,
```

Comment:
They changed it again: 
```suggestion
		GenesisTimestamp:      1695902400,
```